modules/dispatcher/doc/dispatcher_admin.xml

```diff
@@ -1350,6 +1350,20 @@ onreply_route {
 </programlisting>
 		</example>
  	</section>
+	<section id="dispatcher.f.reload">
+		<title>
+		<function moreinfo="none">ds_reload()</function>
+		</title>
+		<para>
+		Reloads the groups and included destinations. The command is
+		disabled for call load based dispatching (algorithm 10) since
```

Comment:
The code doesn't indicate that it really is disabled in this situation. Do you mean "disabled" or "should not be used".